ledger/src/blockstore_processor.rs

```diff
@@ -871,10 +882,12 @@ pub fn process_blockstore_from_root(
         .meta(start_slot)
         .unwrap_or_else(|_| panic!("Failed to get meta for slot {start_slot}"))
     {
+        let replay_tx_thread_pool = create_thread_pool(get_max_thread_count());
```

Comment:
Another pro of this change is that we'll be able to configure this value to be different size for startup and steady-state. That's good because startup involves replaying entire blocks that are already in ledger, so higher amount of parallelism might be warranted vs. steady-state when blocks are gradually streamed over

Reply:
Makes sense. Looks like we're using 1 thread per core already, so opportunity seems to be dialing this down for steady state

Reply:
> so opportunity seems to be dialing this down for steady state

Exactly - already have a good bit of data to support this being the correct decision in https://github.com/anza-xyz/agave/pull/25